app/views/tutors/index.html.erb

```diff
@@ -1,15 +1,51 @@
 <h2>Tutors</h2>
 
-<table>
+<table class="table table-striped">
+  <thead class="thead-inverse">
   <tr>
-    <th>Title</th>
-    <th>Text</th>
+    <th>First Name </th>
+    <th>Last Name </th>
+    <th>Gender </th>
+    <th>Date of Birth </th>
+    <th>Address </th>
+    <th>Home Phone </th>
+    <th>Cell Phone </th>
+    <th>Email Preferred</th>
+    <th>Email Other</th>
+    <th>Native Language </th>
+    <th>Orientation Date </th>
+    <th>Country of Birth</th>
+    <th>Training Date </th>
+    <th>Training Type </th>
+    <th>Referral </th>
+    <th>Education </th>
+    <th>Employment status </th>
+    <th>Specific Occupation </th>
     <th colspan="2"></th>
   </tr>
+  </thead>
 
   <% @tutors.each do |tutor| %>
     <tr>
-      <td><%= tutor.first_name %> <%= tutor.last_name %></td>
+      <td><%= tutor.first_name %></td>
+      <td><%= tutor.last_name %></td>
+      <td><%= tutor.gender%></td>
+      <td><%= tutor.date_of_birth%></td>
+      <td><%= tutor.address1%> <%= tutor.address2 %> <%= tutor.city %>
+        <%= tutor.state %> <%= tutor.zip %></td>
+      <td><%= tutor.home_phone%></td>
+      <td><%= tutor.cell_phone%></td>
+      <td><%= tutor.email_preferred%></td>
+      <td><%= tutor.email_other%></td>
+      <td><%= tutor.native_language%></td>
+      <td><%= tutor.orientation%></td>
```

Comment:
In our main remote repo, in file schema.db line112: ` t.date     "orientation"`
I don't see any` orientation_date`.
Do you have a different local version that is not yet merged?